 #Basic flags every build needs
-LIBS = -lz
+LIBS=-lcrypto -lz -llzma -lbz2 -lcurl -ldeflate -ldl
 CXXFLAGS ?= -g -O3
 CXXFLAGS += -std=c++11 -fopenmp -fsigned-char -D_FILE_OFFSET_BITS=64 #D_FILE_OFFSET_BITS=64 makes nanopolish work in 32 bit systems
 CFLAGS ?= -O3 -std=c99 -fsigned-char -D_FILE_OFFSET_BITS=64 
@@ -50,19 +50,8 @@
     EIGEN_CHECK =
 endif
 
diff --git a/src/nanopolish_call_methylation.cpp b/src/nanopolish_call_methylation.cpp
index 90f2370..029f6aa 100644
--- a/src/nanopolish_call_methylation.cpp
+++ b/src/nanopolish_call_methylation.cpp
@@ -43,7 +43,7 @@
 #include "H5pubconf.h"
 #include "profiler.h"
 #include "progress.h"
-#include "minimap.h"
+#include "../minimap2/minimap.h"
 #include "bseq.h"
 #include "mmpriv.h"
 
diff --git a/src/nanopolish_methyltrain.cpp b/src/nanopolish_methyltrain.cpp
index b97a2f9..436bf11 100644
--- a/src/nanopolish_methyltrain.cpp
+++ b/src/nanopolish_methyltrain.cpp
@@ -45,7 +45,7 @@
 #include "logger.hpp"
 
 #include "nanopolish_scorereads.h"
-#include "Eigen/Dense"
+#include <eigen3/Eigen/Dense>
 
 extern float g_p_skip, g_p_skip_self, g_p_bad, g_p_bad_self;
 
